packages/components/src/tooltip/test/index.js

```diff
@@ -11,60 +11,55 @@ import Tooltip from '../';
  * WordPress dependencies
  */
 import { TOOLTIP_DELAY } from '../index.js';
-import { act } from '@testing-library/react';
 
 describe( 'Tooltip', () => {
 	describe( '#render()', () => {
 		it( 'should render children (abort) if multiple children passed', () => {
-			// Mount: Enzyme shallow does not support wrapping multiple nodes.
-			const wrapper = mount(
+			const { container } = render(
 				<Tooltip>
 					<div />
 					<div />
 				</Tooltip>
 			);
 
-			expect( wrapper.children() ).toHaveLength( 2 );
+			expect( container.children ).toHaveLength( 2 );
 		} );
 
 		it( 'should render children', () => {
-			const wrapper = shallow(
+			render(
 				<Tooltip position="bottom right" text="Help text">
 					<button>Hover Me!</button>
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-			expect( wrapper.type() ).toBe( 'button' );
-			expect( button.children() ).toHaveLength( 1 );
-			expect( button.childAt( 0 ).text() ).toBe( 'Hover Me!' );
+			const button = screen.getByRole( 'button' );
+			expect( button.nodeName ).toBe( 'BUTTON' );
+			expect( button.childNodes ).toHaveLength( 1 );
+			expect( button ).toHaveTextContent( 'Hover Me!' );
 		} );
 
 		it( 'should render children with additional popover when over', () => {
-			const wrapper = shallow(
+			const { container } = render(
 				<Tooltip position="bottom right" text="Help text">
 					<button>Hover Me!</button>
 				</Tooltip>
 			);
 
-			const event = { type: 'focus', currentTarget: {} };
-			wrapper.simulate( 'focus', event );
+			const button = screen.getByRole( 'button' );
+			button.focus();
+			expect( button.nodeName ).toBe( 'BUTTON' );
+			expect( button.childNodes ).toHaveLength( 2 );
+			expect( button.childNodes[ 0 ] ).toHaveTextContent( 'Hover Me!' );
```

Comment:
Feel free to use `data-testid` and use `getByTestId` queries when it's done in tests - that makes it easier to find a certain element without using DOM traversing - that use is fine as long as we don't pollute production code with unnecessary `data-testid` props.

Reply:
I think we can avoid using `data-testid` for this test (see my previous comment on how I would write this test)